addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c

```diff
@@ -14,6 +14,7 @@ modded class SCR_CharacterDamageManagerComponent : SCR_DamageManagerComponent
 	protected ACE_Medical_PainHitZone m_pACE_Medical_PainHitZone;
 	protected float m_fACE_Medical_ModeratePainThreshold;
 	protected float m_fACE_Medical_SeriousPainThreshold;
+	protected float m_fACE_Medical_MedicalKitMaxHeal;
```

Comment:
Settings needs to be replicated, move down to other Rpl props. Also needs a default value in case the settings are not there.
```suggestion
    [RplProp()]
	protected float m_fACE_Medical_MedicalKitMaxHeal;
```